src/features/customerForm/__tests__/CustomerEditForm.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { mount, ReactWrapper } from 'enzyme';
```

Comment:
Should we create a basic test cafe browser test:
1. Open first customer in the list
2. Click edit
2. Check that first name has data
?

Reply:
Sounds good, I'll add it 👍